File: BBIS.Application/DTOs/ApplicationSetting/BloodComponentChecklistDto.cs
namespace BBIS.Application.DTOs.ApplicationSetting
{
    public class BloodComponentChecklistDto
    {
        public Guid? BloodComponentChecklistId { get; set; }
        public Guid BloodComponentId { get; set; }
        public string ChecklistDescription { get; set; }
        public bool IsAdult { get; set; }
        public bool IsActive { get; set; }
    }
}

File: BBIS.Api/ClientApp/src/models/ApplicationSetting/BloodComponentChecklistDto.ts
export interface IBloodComponentChecklistDto {
    BloodComponentChecklistId: Guid;
    BloodComponentId: Guid;
    ChecklistDescription: string;
    IsActive: boolean;
    IsAdult: boolean;
}

export class BloodComponentChecklistDto implements IBloodComponentChecklistDto {
    BloodComponentChecklistId: Guid = '';
    BloodComponentId: Guid = '';
    ChecklistDescription: string = '';
    IsActive: boolean = true;
    IsAdult: boolean = true;
}